lib/flagship.dart

```diff
@@ -70,7 +79,30 @@ class Flagship {
     return _singleton.currentVisitor;
   }
 
-  FlagshipConfig getConfiguration() {
-    return _configuration;
+  /// Get the current configuration
+  FlagshipConfig? getConfiguration() {
+    return _singleton.currentVisitor?.config;
+  }
+
+  ///Active or desactive logger
+  ///
+  /// isLogEnabled : True to activated logger , otherwise false to deactivate
+  static void enableLog(bool isLogEnabled) {
+    LogManager.logEnabled = isLogEnabled;
+  }
+
+  /// Display message logger
+  ///
+  /// level : Level of details for logs
+  /// message : Message to display
+  static void logger(Level level, String message, {bool isJsonString = false}) {
```

Comment:
why logger and not log ?

Reply:
Personally i prefer **logger**  the logger produce log a **printed** message .